views/layouts/auth.hbs

```diff
@@ -0,0 +1,29 @@
+<html lang='en'>
+  <head>
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <link rel='icon' type='image/png' href='favico.png' />
+    <meta charset='utf-8' />
+    <title>{{projectName}}</title>
+    {{! Themes }}
+    <link rel="stylesheet" href="style.css" />
```

Comment:
Do we know the absolute path to the `style.css` here? I always find myself debugging these relative imports if anything changes on the server side, so I prefer absolute imports to make it clear, which file is meant.

Reply:
These same imports are also used on the other files, maybe we could extract the `link` or the whole header to a fragment?

Reply:
since this is an express application, the client side part of this will be the browser loading the `style.css` sheet from the server as a relative URL to whatever the browser url.

e.g. https://custom-domain.com will load the `style.css` as https://custom-domain.com/style.css

The browser handles this loading and this isn't a filesystem load like in nextjs or react.

Reply:
Yeah i think we can maybe move this to a partial

Reply:
Yes, I know^^

But `style.css` is a relative import and those can be hard to debug, because they're relative to the file including them. That's why I prefer absolute imports, e.g. `/some/static/path/styles.css`.

Reply:
but you cannot do absolute imports here since this is an http call back to itself on the root path

Reply:
Eh, absolute paths don't work when the UI is hosted on a path, that's not the root of a domain (e.g. `http://custom-domain.com/auth/login`). Damn, that's annoying. But doesn't matter I guess. LGTM.

Reply:
the express server is explicitly serving the file from the root path.
https://github.com/ory/kratos-selfservice-ui-node/pull/189/files#diff-1f5f84e6c3633404ab2cfd17752c70ae17ff2f973991206a8d2495e3f708674fL7

 we can change this to whatever we want including the `/auth/login` :)